src/modules/createPopup.js

```diff
@@ -0,0 +1,126 @@
+import updateTotalNumberOfReservations, {
+  fetchAllMovieReservations, reservationApi, movieApi,
+} from './assets.js';
+
+const reservationPopUp = document.querySelector('.movie-popup');
+
+const getTvInfo = (tvUrl) => fetch(tvUrl)
+  .then((response) => response.json())
+  .then((actualData) => actualData)
+  .catch((error) => error);
+
+const reservation = (tvUrl, formData = {}) => fetch(tvUrl, {
+  method: 'POST',
+  headers: {
+    'Content-Type': 'application/json',
+  },
+  body: JSON.stringify(formData),
+
+}).then((response) => response.text())
+  .then((actualData) => (actualData.error ? { error: true, info: actualData }
+    : { error: false, info: actualData }))
+  .catch((error) => ({ error: true, info: error }));
```

Comment:
- [ ] It would be nice to use the `async` and `await` keywords with the `fetch` function to get data from the API endpoint according to project requirements. Kindly implement this change. 👍